src/animators/animator.ts

```diff
@@ -1,5 +1,39 @@
 module Plottable {
+export module Animators {
```

Comment:
Typescript allows for declaring nested modules at the top level

Change to `module Plottable.Animators`